common/app/views/fragments/containers/facia_cards/commercialContainer.scala.html

```diff
@@ -104,7 +104,16 @@ <h3 class="paidfor-popup__title">Paid stories are paid for and controlled by an
                                 </div>
                             </div>
                             <h3 class="commercial__title">@container.displayName</h3>
-                            <a class="commercial__cta" href="@LinkTo("/guardian-labs")">@inlineSvg("glabs-logo", "logo")</a>
+                            <a class="commercial__cta"
+                                @if(Edition(request).id == "AU") {
+                                    href="@LinkTo("/guardian-labs-australia")"
+                                } else {
+                                    href="@LinkTo("/guardian-labs")"
+                                    }
+                            >
+                                @inlineSvg("glabs-logo", "logo")
+                                <span class='u-h'>Guardian Labs</span>
```

Comment:
Food for thought: that kind of hidden label should really be in the SVG as a `title` element. Much more accessible and reusable.